config/config-package.py

```diff
@@ -56,6 +71,22 @@ def call(*args, capture_output=False):
 shutil.copy(config_type_path / 'travis.yml', path / '.travis.yml')
 
 
+# Modify templates with meta options.
+tox_ini_path = path / 'tox.ini'
+with open(tox_ini_path) as f_:
+    tox_ini = f_.read()
+
+with open(tox_ini_path, 'w') as f_:
+    fail_under = meta_opts.get('fail-under', '')
+    if fail_under:
+        coverage_report_options = f'--fail-under={fail_under}'
+    else:
+        coverage_report_options = ''
+    f_.write(tox_ini.format(coverage_report_options=coverage_report_options))
+
+with open(meta_cfg_path, 'w') as meta_f:
+    meta_cfg.write(meta_f)
```

Comment:
I think there should a hint in the documentation that `.meta.cfg` is generated if not existing.

Reply:
Done.